src/gui/widget_task.c

```diff
@@ -35,6 +35,7 @@
 #include <LCUI_Build.h>
 #include <LCUI/LCUI.h>
 #include <LCUI/gui/widget.h>
+#include <LCUI/gui/widget_fpsmeter.h>
```

Comment:
Please rename file `widget_fpsmeter.*` to `fpsmeter.*`, Because it is not part of the widget system